JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -16,11 +16,6 @@ module Microsoft.ApplicationInsights.Context {
         public id: string;
 
         /**
-         * The true if this is the first session
-         */
-        public isFirst: boolean;
```

Comment:
I thought that our UI in Settings->Usage->Users->New users chart is based on isFirst flag. Can you double check pls